runtime/kusama/src/lib.rs

```diff
@@ -453,7 +453,7 @@ impl pallet_election_provider_multi_phase::Config for Runtime {
 	type Solver = frame_election_provider_support::SequentialPhragmen<
 		AccountId,
 		pallet_election_provider_multi_phase::SolutionAccuracyOf<Self>,
-		runtime_common::elections::OffchainRandomBalancing,
+		(),
```

Comment:
I assume this is the equivalent to `None::<(usize, sp_npos_elections::ExtendedBalance)>`? What would I look up to confirm this though?

Reply:
[Yes](https://paritytech.github.io/substrate/master/frame_support/traits/trait.Get.html#impl-Get%3CT%3E-for-())